stacks-common/src/util/mod.rs

```diff
@@ -35,6 +35,30 @@ use std::path::Path;
 use std::time::{SystemTime, UNIX_EPOCH};
 use std::{error, fmt, thread, time};
 
+#[cfg(any(test, feature = "testing"))]
```

Comment:
Because this is all test code, can this be placed into a designated test file or module?

Reply:
Can do :D